src/coprocessor/statistics/analyze.rs

```diff
@@ -211,6 +249,8 @@ struct SampleBuilder<S: Snapshot> {
     cm_sketch_depth: usize,
     cm_sketch_width: usize,
     columns_info: Vec<tipb::ColumnInfo>,
+    common_handle_req: Option<AnalyzeIndexReq>,
```

Comment:
Seems this field is useless, if `common_handle_col_ids.len() > 0` then this field is a `Some` value. Otherwise, it is a None or `common_handle_col_ids` is empty.

Reply:
If a table hash common handle. Then `common_handle_col_ids.len() > 0` is always satisfied when analyze columns. But `common_handle_req` only in the mixed type of analyze request.